test/Untitled.ipynb

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Looks like this file got accidentally added?

Reply:
Yup. Good catch.